sonarqube/content.md

```diff
@@ -16,6 +16,8 @@ The server is started this way:
 $ docker run -d --name sonarqube -p 9000:9000 -p 9092:9092 %%IMAGE%%
 ```
 
+The [default username and password](https://docs.sonarqube.org/display/SONAR/Authentication) is `admin:admin`.
```

Comment:
@simon04 nice addition! :thumbsup: To publish I think will be better to phrase this slightly differently
```
By default you can login as `admin` with password `admin`.
```
in order to clearly separate username and password.